model_analyzer/config/input/config_command_profile.py

```diff
@@ -571,6 +583,15 @@ def _add_profile_models_configs(self):
                 " to be used during profiling",
             )
         )
+        self._add_config(
+            ConfigField(
+                "request_period",
+                flags=["--request-period"],
+                field_type=ConfigListNumeric(int),
+                description="Comma-delimited list of request period values or ranges <start:end:step>"
+                " to be used during profiling",
```

Comment:
Request period is unsigned integer scalar value.

Reply:
How is request period different than the other values (concurrency, request rate, batch size) that MA captures?